cmd/sriov/main.go

```diff
@@ -185,6 +185,10 @@ func cmdDel(args *skel.CmdArgs) error {
 
 	sm := sriov.NewSriovManager()
 
+	defer func() {
```

Comment:
In the issue #126, it mentions that device is not visible in init netns until after the last cmdDel finishes, will this defer function be able to detect VF device in the host namespace?

Reply:
It will try to reset the mac on the VF device in the init netns after ReleaseVF() which is called by cmdDel() has renamed it back to its original name and moved it back to the init netns.
One issue I see is if ReleaseVF() didn't succeed in renaming and moving it back to the init netns.

Reply:
You can get the netdevice of the VF by `conf.Master` `conf.VFID` that way you dont rely on its name.

also MAC is restored during release VF. in case of a successful CmdDel (i.e netns was not deleted mid operation)
adding this defer will cause the operation to be performed twice